### ISSUE TITLE
Convert OBJ vertex color into a texture (from command-line)

### ISSUE THREAD

**Issue by AndroYD84** [assimp/assimp#3046]
Hello, I have an OBJ file with vertex colors that I want to preserve, a tool in my pipeline (it's irreplaceable and one of its kind) doesn't support vertex colors and will lose that information after using it, so I need to find some way to put that information back again.
I can do that easily from a GUI tool (ie. Blender, MeshLab, etc.), but my pipeline is headless and requires a command line tool. Is it possible to do that with Assimp? If not, can this feature be implemented?
Assimp can already export textures, so it makes sense someone might want to do the same with vertex colors too.

**Comment by kimkulling** [comment]
So if I understand you right your plan is to generate a texture which stores the vertex-colors, right?

**Comment by AndroYD84** [comment]
That is correct, some software don't support/display vertex colors, so the workaround would be saving that information as a texture.

Exporting the vertex color information (or converting the exported texture back into vertex color) would be useful as well, so it's possible to rollback to using vertex colors again in case that information was lost.

**Comment by tellypresence** [comment]
Any update on this?

**Comment by tellypresence** [comment]
@AndroYD84 Is this still a problem?

**Comment by turol** [comment]
I think this kind of transformation is out of scope for Assimp. You could use our importer/exporter to implement it.
